### PR TITLE
Docker_dev: Build all.

### PR DESCRIPTION
**Pull request by xiaoxq**

### CODE REVIEW

File: docker/build/installers/install_bazel.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")"
 . installer_base.sh
 
 TARGET_ARCH=$(uname -m)
-VERSION="3.2.0"
+VERSION="3.3.0"
```
Comment (storypku) [inline docker/build/installers/install_bazel.sh:27]: Please also update the sha256sum values involved.
Note also for aarch64, we prebuilt bazel  binary of version 3.2.0, and hosted it on bcebos. So either keep version 3.2.0 for aarch64, or build one for 3.3.0 and host it on bcebos.

File: scripts/apollo_lint.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ function run_cpp_lint() {
     _cpp_lint_impl "${targets}"
 
     if [[ "${STAGE}" == "dev" ]]; then
-        _cpp_lint_impl "$(bazel query //modules/... except //modules/tools/visualizer/...)"
+        _cpp_lint_impl "//modules/..."
```
Comment (storypku) [inline scripts/apollo_lint.sh:37]: Good job.

File: docker/build/installers/install_bazel.sh (hunk 1 above)
Comment (xiaoxq) [reply docker/build/installers/install_bazel.sh:27]: done

Comment (xiaoxq) [reply docker/build/installers/install_bazel.sh:27]: For aarch64, I'd like to keep current version and revisit when we have a machine :)

**Comment by xiaoxq** [comment]
@storypku Locked to v3.2 for aarch64. Thanks!

**Pull request merged**